payloads/external/edk2/Kconfig.dasharo

```diff
@@ -363,4 +363,16 @@ config EDK2_USE_UEFIVAR_BACKED_TPM_PPI
 
 	  If unsure, say N.
 
+config EDK2_DISABLE_FAST_BOOT_FEATURE
```

Comment:
I think we want these to be hidden by default

Reply:
Yes, changed: https://github.com/Dasharo/coreboot/pull/626/commits/33a7c96aa0c4823c55484b41d3d31fb890071413 @mkopec